pkgs/applications/editors/ghostwriter/default.nix

```diff
@@ -14,13 +14,13 @@
 
 mkDerivation rec {
   pname = "ghostwriter";
-  version = "2.1.5";
+  version = "2.2.0";
 
   src = fetchFromGitHub {
     owner = "wereturtle";
```

Comment:
Ghostwriter is apparently becoming a KDE project, and the new project URL is <https://github.com/KDE/ghostwriter>, so we can change this to `"KDE"`, and also alter `meta.changelog`.

(Looks like it will eventually [move to the KDE Gitlab](https://github.com/KDE/ghostwriter/issues/821#issuecomment-1257335313), but that's not done yet)

Reply:
Good catch, I added that.